/**:
  ros__parameters:
    config_path: '/home/ponwalai/fibo-technovation/volume_estimate/src/params/lidar_pose.yaml'

lidar_calibrator:
  ros__parameters:
    max_correspondence_distance: 0.1

pointcloud_clustering_node:
  ros__parameters:
    floor_z_height: 0.2
    ceiling_z_height: 8.0
    down_sampling_size: 0.1
    cluster_dbscan_eps: 4.5
    cluster_dbscan_min_points: 50
    cluster_min_points: 100
    normal_weight: 500.0
    normal_estimation_radius: 0.8
    min_pile_angle: 0.0
    max_pile_angle: 55.0

volume_estimator_node:
  ros__parameters:
    voxel_size: 0.5
    floor_height: 0.0
    cluster_dbscan_eps: 1.5
    cluster_dbscan_min_points: 20
    cluster_min_points: 50
    max_interpolation_dist: 3.0